llvm/lib/Target/RISCV/RISCVSubtarget.h

```diff
@@ -138,6 +150,8 @@ class RISCVSubtarget : public RISCVGenSubtargetInfo {
   /// initializeProperties().
   RISCVProcFamilyEnum getProcFamily() const { return RISCVProcFamily; }
 
+  RISCVProfileEnum getRISCVProfile() const { return RISCVProfile; }
```

Comment:
Why do we need profile in Subtarget?

Reply:
I was thinking that we may need it to define ELF attr: https://github.com/riscv-non-isa/riscv-elf-psabi-doc/pull/409. But it seems that we don't need it now.

Reply:
Were you going to remove it?

Reply:
If we use the profile in the backend wouldn't that mean the profile gets different treatment than the equivalent list of extensions?